liquibase-couchbase/src/main/java/liquibase/ext/couchbase/validator/MutateInReplaceValidator.java

```diff
@@ -15,9 +15,8 @@ public MutateInReplaceValidator(MutateInType mutateInType) {
     }
 
     @Override
-    public void validate(String path, Value value, List<Value> values) {
-        validateValuePresence(value);
-        validateNoMultipleValues(values);
```

Comment:
Again we need to ensure that only 1 values is passed. 

You need to create method like validateOneValue where you will check that list contains only 1 value